fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneNotQuery.java

```diff
@@ -91,7 +91,7 @@ public void getPlannerGraphDetails(@Nonnull ImmutableList.Builder<String> detail
 
     @Override
     public int planHash(@Nonnull final PlanHashKind hashKind) {
-        return super.planHash() - PlanHashable.iterablePlanHash(hashKind, negatedChildren);
+        return PlanHashable.iterablePlanHash(hashKind, negatedChildren);
```

Comment:
what about the impact of the regular `children`. Looks like they are not hashed anymore.

Reply:
Good point, we do need this super function after all, but we forgot to provide the hashKind